### DISCUSSION TITLE
Deploy a GitHub branch, not available for this stack

### DISCUSSION THREAD

**Discussion by dessyDS** [teamalgoritma/community#435] [Q&A] [answered]
Hi, mau tanya mengenai eror ini, tidak bisa pakai python versi yang ada di runtime.txt
Failed to compile Python app

![image](https://user-images.githubusercontent.com/112546949/199173308-4d8d5ee9-4f15-45e9-a739-349b0eaee1bf.png)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @dessyDS ,

Permasalahan yang Anda alami disebabkan karena versi python yang digunakan tidak tersedia oleh layanan Heroku. Maka dari itu di file `runtime.txt` file Anda silahkan mengubah ke versi Python yang disupport oleh Heroku, salah satunya adalah `python-3.10.8`. Setelah mengubah versi pada `runtime.txt` silahkan coba deploy kembali. 

Versi python lain yang disupport dapat dilihat pada artikel [ini](https://devcenter.heroku.com/articles/python-support).

Catatan: Anda hanya butuh untuk mengubah `runtime.txt` tanpa harus menginstall/mengupdate versi Python pada komputer Anda.

Jika jawaban ini dirasa membantu, silahkan tandai sebagai "Mark As Answer".
